File: c99/beecrowd_1397.c
#include <stdio.h>

int main()
{

    int n, a, b, count_a, count_b;

    while (1)
    {
        scanf("%d", &n);

        if (n == 0)
        {
            break;
        }

        count_a = count_b = 0;

        for (int i = 0; i < n; i++)
        {
            scanf("%d %d", &a, &b);
            if (a > b)
            {
                count_a++;
            }
            else if (b > a)
            {
                count_b++;
            }
        }

        printf("%d %d\n", count_a, count_b);
    }
    return 0;
}

File: 2-AD-HOC/python/beecrowd_1397.py
while True:
    n = int(input())

    if n == 0:
        break

    count_a = count_b = 0

    for i in range(n):
        a, b = map(int, input().split())

        if a > b:
            count_a += 1
        elif b > a:
            count_b += 1
            
    print(count_a, count_b)
